Tests/StoreKitUnitTests/StoreKitConfigTestCase.swift

```diff
@@ -20,7 +20,8 @@ import XCTest
 @available(iOS 14.0, tvOS 14.0, macOS 11.0, watchOS 6.2, *)
 class StoreKitConfigTestCase: TestCase {
 
-    static var requestTimeout: DispatchTimeInterval = .seconds(60)
+    static var requestTimeout: TimeInterval = 60
+    static var requestDispatchTimeout = DispatchTimeInterval.seconds(Int(requestTimeout))
```

Comment:
I think this should be a derived property, so that it responds to changes to `requestTimeout`, right?
```suggestion
    static var requestDispatchTimeout: DispatchTimeInterval {
      return .seconds(Int(Self.requestTimeout))
    }
```

Reply:
Oh, good call. ✅